interface/src/app/gene/network/network.less

```diff
@@ -41,15 +41,11 @@ text.legend-title {
 }
 
 .edge-tip {
-  #title {
-    text-align: center;
-  }
-
   line-height: 1;
   font-weight: bold;
   padding: 12px;
-  background: rgba(100, 150, 200, 0.9);
-  color: #fff;
+  background: Gainsboro;
```

Comment:
Awesome. I don't think I've ever seen anyone use color names other than `black`, `white`, and maybe `red`.